src/parser/model/organization-root-resource.ts

```diff
@@ -6,25 +6,32 @@ export interface IOrganizationRootProperties {
     ServiceControlPolicies: IResourceRef | IResourceRef[];
     DefaultOrganizationAccessRoleName?: string;
     DefaultBuildAccessRoleName?: string;
+    MirrorInGovCloud?: boolean;
```

Comment:
pffff this is an interesting question.... should the --partition be leading or something in the organizationRoot?
somehow i think it makes sense to have both and throw if the values dont match. just to make sure the arguments are passed properly (not forgotten/changes).

so:
```
  OrganizationRoot:
  Type: 'ORG:OrganizationRoot'
  Properties:
     MirrorPartition: 'us-gov'
```

will cause an error when:
- both `--partition` and `--mirror-partition` are not specified
- either `--partition` or `--mirror-partition` is specified but has a value other than specified in the OrganizationRoot (e.g. value is `eu-gov` instead of `us-gov` or `us_gov` instead of `us-gov`


Lets add validation on the MirrorPartition property to only accept `us-gov`.
I am not sure what to do when the MirrorPartition value is changed. probably not do anything for now (at your own risk). only possible values would be (unspecified and `us-gov`)